### PR TITLE
Homework2

### PR DESCRIPTION
**Pull request by ayshajamjam**

### CODE REVIEW

File: students/aj2604.md [hunk 1]
```diff
@@ -6,15 +6,10 @@ UNI: aj2604
 
 Github: [link](https://github.com/ayshajamjam)
 
-## Project Proposal
+GitHub README: [link](https://github.com/ayshajamjam/ayshajamjam/blob/main/README.md)
 
-I want to make a chrome extension that can parse through your screen (scrape the text) and determine which vocabulary words you may be unfamiliar with. This can be particularly useful when reading articles online. If you click the extension's button it would then show you the list of words and the definition underneath. You can also save words for future reference.
+## Project Proposal
+- [vocab-extension](./projects/javascript/vocab-extension.md)
```
Comment (timkpaine) [inline students/aj2604.md:12]: update path to `../`

File: students/aj2604.md [hunk 2]
```diff
@@ -6,15 +6,10 @@ UNI: aj2604
...
 
 ## Project Contributions
-
-1. NumPy
-2. [MapTalks](https://github.com/maptalks)
-3. [SpaceTime](https://github.com/spencermountain/spacetime)
-4. [Glide](https://github.com/bumptech/glide)
-5. [Stanford-Oval](https://github.com/stanford-oval)
-6. [FreeCodeCamp](https://github.com/freeCodeCamp/freeCodeCamp/issues)
+- [contribution](./projects/python/contribution.md)
```
Comment (timkpaine) [inline students/aj2604.md:15]: update path to `../`

**Review by timkpaine — CHANGES_REQUESTED** [review]
ensure commits are squashed into 1

File: students/aj2604.md (hunk 1 above)
Comment (ayshajamjam) [reply students/aj2604.md]: Done

File: students/aj2604.md (hunk 2 above)
Comment (ayshajamjam) [reply students/aj2604.md]: Done

**Pull request merged**